src/alttxt/parser.py

```diff
@@ -150,20 +150,25 @@ class should be implemented to parse aggregated data.
             # size
             size: int = int(item.get("size", self.default_field))
             # Deviation - rounded to 2 decimals
-            dev: float = round(item.get("deviation", self.default_field), 2)
+            # Backwards compatibility (<v0.2.7)
+            try:
+                dev = item.get("attributes", self.default_field)["deviation"]
```

Comment:
Looks like deviation is still unrounded in the data export, no? It should be rounded here so that the alttxt doesn't include needlessly long numbers in the output.